src/basics/Player.js

```diff
@@ -121,6 +123,7 @@ export class Player {
 			return !linked_orders.has(c.order) &&
 				card.possibilities.every(p => (card.chop_moved ? state.isBasicTrash(p) : false) || state.isPlayable(p)) &&	// cm cards can ignore trash ids
 				card.possibilities.some(p => state.isPlayable(p)) &&	// Exclude empty case
+				(options?.assume || !card.self_connection) &&
```

Comment:
If you want the default to be `{ assume: true }`, then I think it's more robust to use `(options?.assume ?? true)` here and without any default parameters in `thinksLoaded()` or `thinksPlayables()`. Otherwise, if a second option gets added, then calling such functions with only the second option will skip default assignment and result in `assume` being false.

Reply:
Thanks, I think I've got this done as suggested.

I'm not exactly sure that assuming true is the right default, but I'm not sure that it's not, worth thinking more about particular cases I guess. I've explicitly added the assume value in the cases where it's important, but in the cases where it's not specified tests will pass with either true or false as the default fallback currently.